config/environments/production.rb

```diff
@@ -59,16 +61,29 @@
   # config.active_job.queue_name_prefix = "project_name_snake_production"
 
   config.action_mailer.perform_caching = false
-
   config.action_mailer.delivery_method = :smtp
-  config.action_mailer.smtp_settings = {
-    address: "smtp-relay.sendinblue.com",
-    port: 587,
-    user_name: ENV.fetch("SENDINBLUE_EMAIL", nil),
-    password: ENV.fetch("SENDINBLUE_PASSWORD", nil),
-    authentication: "login",
-    enable_starttls_auto: true
-  }
+
+  if ENV.key?("SENDINBLUE_USERNAME")
+    config.action_mailer.smtp_settings = {
+      address: "smtp-relay.sendinblue.com",
+      port: 587,
+      user_name: ENV.fetch("SENDINBLUE_EMAIL"),
+      password: ENV.fetch("SENDINBLUE_PASSWORD"),
+      authentication: "login",
+      enable_starttls_auto: true
+    }
+  elsif ENV.key?("MAILTRAP_API_TOKEN")
+    response = URI.parse("https:///api/v1/inboxes.json?api_token=#{ENV.fetch('MAILTRAP_API_TOKEN')}").read
+    first_inbox = JSON.parse(response)[0]
```

Comment:
Smart. Is this from the documentation?

Reply:
Yes, but I altered it slightly to not require a third party gem.